UI/EnemyFileMenu.gd

```diff
@@ -209,8 +209,16 @@ func _do_load_file_json(file: File) -> int:
 		var enemy_set = SetProvider.get_enemy_set(stage_id, layer_no, group_id, subgroup_id)
 		enemy_set.add_enemy(enemy)
 		
-		if enemies_schema_idx.has("SpawnTimeId"):
-			enemy.time_type = data[enemies_schema_idx["SpawnTimeId"]]
+		if enemies_schema_idx.has("SpawnTime"):
+			var time_range_str = data[enemies_schema_idx["SpawnTime"]]
+			var time_type = 0
+			if time_range_str == "00:00,23:59":
+				time_type = 0
+			elif time_range_str == "07:00,18:00":
+				time_type = 1
+			elif time_range_str == "18:00,07:00":
```

Comment:
If the ranges are inclusive, at precisely 18:00 both enemies for day and night will show up. It may be better to have it as 07:00,17:59 and 18:00,06:59 instead

Reply:
If it doesn't match any of the three time types we support, print a warning and set to all day (for example)

Reply:
a 3 second window is still a window, it shouldn't happen at all, considering how easy of a change this is, it should be fixed. About the second point, It should also print a warning instead of silently dropping data